mmdet3d/models/dense_heads/ssd_3d_head.py

```diff
@@ -56,7 +56,9 @@ def __init__(self,
                  dir_res_loss=None,
                  size_res_loss=None,
                  corner_loss=None,
-                 vote_loss=None):
+                 vote_loss=None,
+                 velocity_loss=None):
+        self.extra_reg_dim = 2 if velocity_loss is not None else 0
```

Comment:
Can we take the velocity as one of our regression targets of bounding boxes? It is the typical approach in anchor-based methods.